package.json

```diff
@@ -23,10 +23,12 @@
     "@graphql-tools/links": "6.2.5",
     "@graphql-tools/stitch": "6.2.4",
     "@graphql-tools/utils": "6.2.4",
+    "@node-rs/bcrypt": "1.1.0",
     "@parse/fs-files-adapter": "1.2.0",
     "@parse/push-adapter": "3.4.0",
     "@parse/s3-files-adapter": "1.6.0",
     "@parse/simple-mailgun-adapter": "1.2.0",
+    "@simplewebauthn/server": "^2.2.0",
```

Comment:
Remove range operator

Reply:
After a long search on the web, implementing a webauthn server (and also a front) handler on our side... will be a nightmare. This lib is well coded by Masterkale (i contributed to this pkg also for some minor fixes). Webauthn follow a fixed W3C spec; currently Masterkale and some other contributors keep it synced correctly.

In node/npm eco system from my point of view this the best lib to implement this feature correctly with a maximal device compatibility.